spec/unit/provider/user/linux_spec.rb

```diff
@@ -69,7 +69,7 @@
 
     it "throws a deprecation warning on setting supports[:manage_home]" do
       Chef::Config[:treat_deprecation_warnings_as_errors] = false
-      expect(Chef).to receive(:deprecated).with(:supports_property, "supports { manage_home: true } on the user resource is deprecated and will be removed in Chef 13, set manage_home: true instead")
+      expect(Chef).to receive(:deprecated).with(:supports_property, "supports { manage_home: true } on the user resource is deprecated and will be removed in Chef 13, set manage_home true instead")
```

Comment:
Also if I can be slightly pedantic it would have to be `supports({ manage_home: true })` because lolrubby :(